core/modules/editor/operations/text/wrap-lines.js

```diff
@@ -13,8 +13,8 @@ Text editor operation to wrap the selected lines with a prefix and suffix
 "use strict";
 
 exports["wrap-lines"] = function(event,operation) {
-	var prefix = operation.paramObject.prefix || "",
-		suffix = operation.paramObject.suffix || "";
+	var prefix = event.paramObject.prefix || "",
```

Comment:
Might we need to check for event.paramObject being undefined?

Reply:
@Jermolene I considered that but it should not be needed, see https://github.com/Jermolene/TiddlyWiki5/blob/9fce8153dfdd5e2378f1827425c5b1ebbd265df1/core/modules/widgets/action-sendmessage.js#L61-L65

Reply:
Great thanks @saqimtiaz